includes/Support/Path.php

```diff
@@ -15,12 +15,16 @@ public static function temporary(): Path
         return new Path(sys_get_temp_dir());
     }
 
-    public function to($path = ""): string
+    public function to(string $subpath): string
     {
-        if (!strlen($path)) {
+        if (!strlen($subpath)) {
             return $this->basePath;
         }
+        return $this->basePath . DIRECTORY_SEPARATOR . ltrim($subpath, "/");
```

Comment:
Shouldn't we use `DIRECTORY_SEPARATOR` instead of `"/"`?

Reply:
I thought we would do it in a separate PR